### PR TITLE
#165305766 List users functionality

### PR DESCRIPTION
**Pull request by mdn-network**
#### Description
Lists all users
#### Type of change
- [ ] Bug fix (nonbreaking change which fixes an issue)
- [x] New feature (nonbreaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### How Has This Been Tested?
Please describe the tests that you ran to verify your changes
- [ ] End to end
- [ ] Integration

#### Checklist:
- [ ] My code follows the style guide for this project
- [ ] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[165305766](https://www.pivotaltracker.com/story/show/165305766)

### CODE REVIEW

**Review by a-braham — COMMENTED** [review]
You could add the url for getting all users in the read me file

File: authors/apps/profiles/urls.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from authors.apps.profiles.views import (UserProfileView, UpdateUserProfileView, UserListView)
+
+app_name = 'profiles'
+
+urlpatterns = [
+    path('profiles/<str:username>', UserProfileView.as_view(), name='profile'),
+    path('profiles/<str:username>/', UpdateUserProfileView.as_view(), name='update_profile'),
+    path('profiles/', UserListView.as_view(), name='list_users'),
```
Comment (Ogutu-Brian) [inline authors/apps/profiles/urls.py:17]: Hello @Philipotieno ,
Add tests for this endpoint. It is currently untested

**Review by a-braham — COMMENTED** [review]
You do not need to commit the migrations file since it is handled at @a-braham PR

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

File: authors/apps/profiles/urls.py (hunk 1 above)
Comment (mdn-network) [reply authors/apps/profiles/urls.py:17]: Just added the tests @Ogutu-Brian

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Review by matthenge — COMMENTED** [review]
Please fix the coverage drop so that the PR can be merged

**Review by matthenge — APPROVED** [review]
LGTM

**Review by a-braham — APPROVED** [review]
LGTM

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Pull request merged**